src/main/java/org/jabref/gui/push/PushToApplicationsManager.java

```diff
@@ -25,15 +25,15 @@ public PushToApplicationsManager(DialogService dialogService) {
         return applications;
     }
 
-    public static PushToApplicationSettings getSettings(PushToApplication application) {
+    public PushToApplicationSettings getSettings(PushToApplication application, DialogService dialogService) {
         if (application instanceof PushToEmacs) {
```

Comment:
And now you can even go one step further and remove the parameter DialogService here and  do the following:
```
 public PushToApplicationsManager(DialogService dialogService) {
this.dialogService = dialogService
```

Reply:
already did, checks just told me too- ;-)